src/components/ContestTile.js

```diff
@@ -55,27 +54,30 @@ const ContestTile = ({ contest: { node }, updateContestStatus }) => {
           />
         ) : (
           <p>
-            Contest ran {t.startDay}-{t.endDay}
+            Contest ran {t.startDay === "" ? "" : t.startDay}-
+            {t.endDay === "" ? "" : t.endDay}
           </p>
         )}
         <ClientOnly>
           <Link
-            to={contestPath}
+            to={fields?.contestPath || "/"}
             className="contest-repo button button-small cta-button primary"
           >
             {`${findingsRepo === "" ? "Preview" : "View"} Contest`}
           </Link>
           {t.contestStatus === "active" && contestRepo && (
-            <a
-              href={contestRepo}
+            <Link
```

Comment:
The `Link` tag is for internal links -- https://www.gatsbyjs.com/docs/reference/built-in-components/gatsby-link/#reminder-use-link-only-for-internal-links

Reply:
Oh didn't know ! Thanks